go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/docker/docker v1.13.1
 	github.com/docker/go-connections v0.4.0 // indirect
 	github.com/docker/go-units v0.4.0 // indirect
+	github.com/go-delve/delve v1.4.0 // indirect
```

Comment:
Why is this here? It seems to have created a huge mess in `go.sum`. The only thing that should change in this PR is `prep`.